deno.jsonc

```diff
@@ -34,7 +34,9 @@
     "dnt": "deno task run _tasks/dnt.ts",
     "test": "deno task capi serve -- deno test -A -L=info --ignore=target --parallel -r=http://localhost:4646/",
     "test:update": "deno task test -- --update",
-    "test:eg": "deno task run https://deno.land/x/egts@v0.1.0-beta.3/test.ts 'examples/**/*.eg.ts'",
+    "test:eg": "deno task run https://deno.land/x/egts@v0.1.0-beta.5/main.ts",
+    "test:eg:deno": "deno task test:eg test deno 'examples/**/*.eg.ts'",
+    "test:eg:node": "deno task test:eg test node 'target/npm/capi-examples/esm/examples/**/!(metadata)*.eg.js'",
```

Comment:
Rather than having this exclude hardcoded here, perhaps the `@test_skip` directive can take a scope, like `@test_skip node`.